pkg/sync/adapter.go

```diff
@@ -173,6 +170,37 @@ func (a Adapter) SyncFiles(syncParameters common.SyncParameters) (isPushRequired
 	return true, nil
 }
 
+// TODO remove if no required
```

Comment:
what does this mean?

Reply:
I meant `if not required`. Currently the PR is `WIP` so I am testing out certain things.